deepseek-r1 Notebook: Add Option for DeepSeek-R1-Distill-Qwen-32B Model

ekurniaw opened the pull request:
- https://github.com/openvinotoolkit/openvino_notebooks/issues/2718
 - The 32B is the one comparable in capability as OpenAI-o1-mini 100B.
 - Tested with OpenVINO 2024.6 on ARL-S 285K CPU with 32 GB RAM and 500 GB drive (200 GB for vRAM swap file and 200 GB to store the model). Also, tested successfully on ARL-H 285H iGPU with 64 GB RAM.
 - Tested with OpenVINO 2025.0 on ARL-S 285K CPU as well.

ekurniaw committed:
deepseek-r1 Notebook: Add Option for DeepSeek-R1-Distill-Qwen-32B Model
 - The 32B is the one comparable in capability as OpenAI-o1-mini 100B.

eaidova commented:
@ekurniaw could you please also update notebook description and readme with  model info? Also it is good to mention about RAM requirements

ekurniaw committed:
Merge pull request #1 from openvinotoolkit/latest

Rebase

ekurniaw committed:
Update deepseek-r1 Notebook with New Model Information
 - Add DeepSeek-R1-Distil-Qwen-32B model information to deepseek-r1 Notebook and README.md.

ekurniaw committed:
Merge branch 'latest' into latest

ekurniaw committed:
Merge pull request #2 from ekurniaw/deepseek-r1-qwen-32b

deepseek-r1 Notebook: Add Option for DeepSeek-R1-Distill-Qwen-32B Model

ekurniaw committed:
Fix deepseek-r1 Notebook Spell Check
 - Remove vRAM word.

ekurniaw committed:
Merge pull request #3 from ekurniaw/deepseek-r1-qwen-32b

Fix deepseek-r1 Notebook Spell Check
